README.md

```diff
@@ -2,6 +2,10 @@
 
 This code allows you to fetch all transactions from UTXO based blockchains to kafka.
 
+## Setup
+
+Because all data is fetched from public end-point, so no need to use k8s proxy or vpn to run the service locally.  
```

Comment:
Are you sure about this? The service needs a full node, so it will be needed to have an access to this. The easiest for us is to connect to the VPN and use the full nodes that we run.

Reply:
Oops, missed it. Will be fixed.